javaslang-benchmark/src/test/java/javaslang/JmhRunner.java

```diff
@@ -0,0 +1,118 @@
+package javaslang;
+
+import javaslang.collection.Array;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.results.RunResult;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.options.*;
+
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+import java.util.function.*;
+
+public class JmhRunner {
+    public static boolean WITH_ASSERTS = false;
```

Comment:
Shouldn't this be `withAsserts` instead of `WITH_ASSERTS`?
I find it misleading, as I would expect upper case for constants only.